src/pipeformer/internal/util.py

```diff
@@ -0,0 +1,192 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Additional utilities."""
+import logging
+import time
+from typing import Dict
+
+import attr
+import awacs.iam
+import awacs.s3
+from attr.validators import deep_mapping, instance_of
+from botocore.exceptions import ClientError
+from troposphere import AWS_ACCOUNT_ID, AWS_PARTITION, AWS_REGION, AWSObject, Sub
+
+from pipeformer.identifiers import LOGGER_NAME
+
+VALUE_SEPARATOR: str = "0"
```

Comment:
`util` or `constants` (or similar)? Or is this definitely not used outside of this file scope?

Reply:
Moved into `identifiers`.